apps/protocol-frontend/src/hooks/useDisplayName.ts

```diff
@@ -9,18 +9,17 @@ const useDisplayName = () => {
 
   const { data: userEnsName, isLoading } = useEnsNameEthers(address);
 
-  if (isLoading) {
-    return { displayName: null };
-  }
-
   const displayName =
     userData?.name ||
     userData?.display_name ||
     userDataByAddress?.name ||
-    userDataByAddress?.display_name ||
-    userEnsName ||
-    displayAddress(address);
-  return { displayName };
+    userDataByAddress?.display_name;
```

Comment:
Not sure if it's only me, but this is delaying the render of the entire connect wallet component while it resolves (comparing to what we currently have on prod)

I'll continue to test this